05LessonFive/Checkpoint1/css/style.css

```diff
@@ -1,43 +1,51 @@
-.lawn {
-    width: 100%;
-    height: 300px;
+#bothboxes img { /*Container with both boxes*/
+  width:500px;
+  float:left;
+  display: inline-block;
+  position:relative;
 }
 
```

Comment:
consider setting the width of the container to that pixel and the having the image width be 100%.